plot-api/src/commonMain/kotlin/jetbrains/letsPlot/FrontendContext.kt

```diff
@@ -8,4 +8,20 @@ package jetbrains.letsPlot
 interface FrontendContext {
     fun display(plotSpecRaw: MutableMap<String, Any>)
     fun getInfo() = "Not available"
-}
\ No newline at end of file
+}
+
+internal fun undefinedFrontendContext() = object : FrontendContext {
+    override fun display(plotSpecRaw: MutableMap<String, Any>) {
+        throw IllegalStateException(
+            """
+                            
+                            The frontend context is not defined.
+                            To define the frontend context please select one of the following options:
+                            a. Add "lets-plot-batik-<version>.jar" to your classpath.   
```

Comment:
Suggestions a) b) make no sense for JS target.

Reply:
Done